x/application/types/errors.go

```diff
@@ -3,10 +3,13 @@ package types
 // DONTCOVER
 
 import (
-	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	errorsmod "cosmossdk.io/errors"
 )
 
 // x/application module sentinel errors
 var (
-	ErrSample = sdkerrors.Register(ModuleName, 1100, "sample error")
+	ErrAppInvalidStake   = errorsmod.Register(ModuleName, 1, "invalid application stake")
+	ErrAppInvalidAddress = errorsmod.Register(ModuleName, 2, "invalid application address")
+	ErrAppUnauthorized   = errorsmod.Register(ModuleName, 3, "unauthorized application")
```

Comment:
When I first read this, I misinterpreted the meaning as there is ambiguity around the word "application". After looking at the usages, it's clear that "application" is referring to the message signer (whom we're expecting to be an application) and not the application whose on-chain state the message is attempting to modify. 

Wdyt about `unauthorized signer`?:
```suggestion
	ErrAppUnauthorized   = errorsmod.Register(ModuleName, 3, "unauthorized signer")
```